### PR TITLE
Some minor Theme color fixes

### PR DESCRIPTION
**Pull request by berndsi**
Also added another article to the Theme demo.

### CODE REVIEW

File: src/transform/ThemeTransform.js [hunk 1]
```diff
@@ -41,11 +41,18 @@ const setTheme = (document, theme) => {
...
-const footballTemplateImageFilenameRegex =
-  new RegExp('Kit_(body|socks|shorts|right_arm|left_arm)(.*).png$')
+const footballTemplateImageFilenameRegex = /Kit_(body|socks|shorts|right_arm|left_arm)(.*).png$/
+
+/**
+ * Dialog image filename regex.
+ * https://commons.wikimedia.org/wiki/Comparison_of_icon_sets
+ * @type {RegExp}
+ */
+const dialogImageFilenameRegex = /-[Dd]ialog-\w+\.svg(\.png)?$/
```
Comment (niedzielski) [inline src/transform/ThemeTransform.js]: You might want to make this whole regex case insensitive like `/-dialog-\w+\.svg(\.png)?$/i`

File: src/transform/ThemeTransform.js [hunk 2]
```diff
@@ -80,13 +90,16 @@ const classifyElements = element => {
...
+  /* en > Wikipedia:How_to_read_a_color_infobox > 762137635 */
   const selector = [
     'div.color_swatch div',
     'div[style*="position: absolute"]',
     'div.barbox table div[style*="background:"]',
     'div.chart div[style*="background-color"]',
     'div.chart ul li span[style*="background-color"]',
     'span.legend-color',
+    'table span[style*="background-color"]',
+    'table td[style*="background-color"]',
```
Comment (niedzielski) [inline src/transform/ThemeTransform.js]: We've got so many of these background attribute selectors now that I wonder if we can generalize them.

File: src/transform/ThemeTransform.js (hunk 1 above)
Comment (berndsi) [reply src/transform/ThemeTransform.js]: Done

File: src/transform/ThemeTransform.js (hunk 2 above)
Comment (berndsi) [reply src/transform/ThemeTransform.js]: Not sure.

**Comment by montehurd** [comment]
Hey Bernd! I reviewed the theme demo articles with your PR and found a couple small regressions:

| Before this PR  | After this PR |
| ------------- | ------------- |
|manchester united f c|manchester united f c 2|
|manchester united f c 3|manchester united f c 4|
||default|

**Comment by berndsi** [comment]
I'm not working on this any time soon.

**Pull request closed without merging**